tutorcelery/patches/k8s-deployments

```diff
@@ -84,4 +84,30 @@ spec:
             value: "5555"
           - name: FLOWER_BASIC_AUTH
             value: {{CELERY_FLOWER_BASIC_AUTH}}
+
+{% if CELERY_FLOWER_SERVICE_MONITOR -%}
```

Comment:
```suggestion
{% if CELERY_FLOWER and CELERY_FLOWER_SERVICE_MONITOR -%}
```

Reply:
This if statement is nested on a `{% if CELERY_FLOWER %}` statement already